bin/hdfcoinc/pycbc_page_foundmissed

```diff
@@ -71,12 +68,31 @@ vals['mtotal'] = m1 + m2
 vals['mass_ratio'] = numpy.maximum(m1/m2, m2/m1)
 
 dvals = {}
-if not args.ifos:  # behaviour with hard-coded H1L1 search for backwards compatibility
-    hdist = f['injections/eff_dist_h'][:]
-    ldist = f['injections/eff_dist_l'][:]
-    dvals['decisive_distance'] = numpy.maximum(hdist, ldist)
-    dvals['dec_chirp_distance'] = pycbc.pnutils.chirp_distance(
-                                                 dvals['decisive_distance'], vals['mchirp'])
+
+ifos = f.attrs['ifos'].split(' ')
+
+# NOTE: Effective distance is hardcoded to these values. It's also normally
+#       meaningless for precessing injections.
+eff_dist_map = {}
+eff_dist_map['H1'] = 'eff_dist_h'
+eff_dist_map['H2'] = 'eff_dist_h'
+eff_dist_map['L1'] = 'eff_dist_l'
+eff_dist_map['G1'] = 'eff_dist_g'
+eff_dist_map['T1'] = 'eff_dist_t'
+eff_dist_map['V1'] = 'eff_dist_v'
+
+try:
+    eff_dists = [f['injections/{}'.format(eff_dist_map[ifo])][:]\
+                 for ifo in ifos]
+    eff_dists = numpy.array(eff_dists).T 
+    dvals['decisive_distance'] = numpy.sort(eff_dists)[:,1]
```

Comment:
add comment eg 'decisive distance is second smallest effective distance'

Reply:
Added